doc/sphinx-guides/source/user/dataset-management.rst

```diff
@@ -99,23 +99,23 @@ rsync is typically used for synchronizing files and directories between two diff
 File Upload Script
 ~~~~~~~~~~~~~~~~~~
 
-An rsync-enabled Dataverse installation has a file upload process that differs from the traditional browser-based upload process you may be used to. In order to transfer your data to Dataverse's storage, you will need to complete the following steps:
+An rsync-enabled Dataverse installation has a file upload process that differs from the traditional browser-based upload process you may be used to. In order to transfer your data the to Dataverse installation's storage, you will need to complete the following steps:
```

Comment:
Misplaced "the", should go after the "to" not before.

Reply:
fixed in https://github.com/IQSS/dataverse/pull/7567/commits/f41355f317062479dce6cb4d02f716459681e8cc